app/models/check_callbacks.rb

```diff
@@ -2,7 +2,9 @@
 
 class CheckCallbacks
   def after_create(check)
-    NotificationFacade.checked(check) if check.sender != check.receiver && check.checkable_type != 'Report'
+    if check.sender != check.receiver && check.checkable_type != 'Report'
+      ActivityDelivery.with(sender: check.sender, receiver: check.receiver, check: check).notify(:checked)
```

Comment:
```suggestion
      ActivityDelivery.with(
        sender: check.sender,
        receiver: check.receiver,
        check: check
      ).notify(:checked)
```
こんな感じに書くと見易いかなと思いました！